packages/accordion-react/README.md

```diff
@@ -1,8 +1,8 @@
-# [`@fremtind/jkl-accordion-react`](https://fremtind.github.io/jokul/components/accordion/)
+# [`@fremtind/jkl-accordion-react`](https://fremtind.github.io/jokul/accordion-react/documentation/accordion/)
 
 ## Beskrivelse
 
-[Bruk og prinsipper](https://fremtind.github.io/jokul/components/accordion/) er beskrevet i `@fremtind/jkl-accordion` stil-pakken.
+[Bruk og prinsipper](https://fremtind.github.io/jokul/accordion-react/documentation/accordion/) er beskrevet i `@fremtind/jkl-accordion` stil-pakken.
```

Comment:
Du får erstattet samtlige av disse med følgende regex i VSCode (jeg ble for nyskjerrig 😅):

find:
```
\[Bruk og prinsipper\]\(https://fremtind.github.io/jokul/([\w-]+)-react/documentation/(\w+)/\) er beskrevet i `@fremtind/jkl-[\w-]+` stil-pakken.
```
replace:
```
Se portalen for [bruk og prinsipper](https://fremtind.github.io/jokul/$1-react/documentation/$2/).
```